roles/rbac/basic_user_role_prereq/tasks/main.yml

```diff
@@ -0,0 +1,68 @@
+- name: Prereq role
+  when: "{{ lookup('ENV', 'RBAC_PREREQ') }}"
+  block:
+    - name: Create user with htpasswd file
+      htpasswd:
+        path: "{{ master_config_dir }}/htpasswd"
+        name: testuser1
+        password: test
+      become: true
+
+    - name: Create user with htpasswd file
+      htpasswd:
+        path: "{{ master_config_dir }}/htpasswd"
+        name: testuser2
+        password: test
+      become: true
```

Comment:
@alexander-demichev We could use a with_items loop here with a list, since the settings for these are identical, just username are different.